docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -9,12 +9,12 @@ like Oracle and Hive, or different Oracle database instances.
 Requirements
 ------------
 
-Requirements for using the connector in a catalog to connect to a Oracle data
-source are:
+To connect to an Oracle data source using the connector in a catalog, you
+need:
 
-* Oracle 12 or higher
-* Network access, by default on port 1521, from the Trino coordinator and
-  workers to Oracle.
+* Oracle 12 or higher. 
+* Network access from the Trino coordinator and workers to Oracle. 
+  Port 1521 is the default port and is used in the code examples.
```

Comment:
Not sure why this appears to be indented

Reply:
Meant to be a continuation of the second bullet's paragraph, right?

Reply:
Correct, it is a continuation of the bullet above.